app/code/Magento/ProductAlert/README.md

```diff
@@ -1 +1,43 @@
-The Magento_ProductAlert module enables product alerts, which allow customers to sign up for emails about product price or stock status change.
+# Magento_ProductAlert module
+
+This module enables product alerts, which allow customers to sign up for emails about product price or stock status change.
+
+## Installation
```

Comment:
Please add some information about the schema/column not being persistent (e.g. not having to be manually removed) when the module gets disabled and setup:upgrade is run.
Additionally, please add information about the recurring script in the module and if its modification need to be manually reverted upon uninstallation.